### PR TITLE
merge inh/act into feature

### LINKED ISSUE #137: Merge INH/ACT into one value
- [x] leave one formula as they're the same
- [x] select INH/ACT at the beginning of the screening process (only for the naming purposes)
- [x] add unusual activation formula as an optional one

### PR DESCRIPTION
**Pull request by azywot**
Resolves #137.

Now the user can specify the screening feature (ACT/INH) and choose the appropriate formula for the activation calculation:
obraz

Based upon that, the summary entry displays feature columns (as we discussed, the computational part remains the same, it's just a change to the name):
obraz

Pasting here the comparison between `ACTIVATION` formulas as well:
obraz

**NOTE: the output differs respectively i.e. one will have either `% ACTIVATION` or `% INHIBITION`**
Any suggestions are welcome 🙏🙏

### CODE REVIEW

**Comment by zuzg** [comment]
So with this approach, if I wanted to get results for inhibition and altered activitation (like in the example in our notes from KŻ), I would have to run the process twice?

**Comment by azywot** [comment]
> So with this approach, if I wanted to get results for inhibition and altered activitation (like in the example in our notes from KŻ), I would have to run the process twice?

Yes, to have it merged into one feature I assumed that during the screening we care about either inhibition or activation. It can be altered to consider both, though if the output of the screening part serves as an input to the next stages, I think it's easier to just select the column containing `%`.

File: dashboard/pages/screening/callbacks.py [hunk 1]
```diff
@@ -654,94 +682,78 @@ def register_callbacks(elements, file_storage):
...
-        State("activation-plot", "figure"),
+        Output("feature-plot", "figure", allow_duplicate=True),
+        Output("feature-button", "disabled"),
+        Input("feature-min-input", "value"),
+        Input("feature-max-input", "value"),
+        State("feature-plot", "figure"),
+        State("activation-inhibition-screening-options", "data"),
         prevent_initial_call=True,
-    )(functools.partial(on_range_update, key="% ACTIVATION"))
+    )(functools.partial(on_range_update))
```
Comment (Tremirre) [inline dashboard/pages/screening/callbacks.py:739]: partial not needed here (no args are filled), it's sufficient to pass the `on_range_update` func itself

File: dashboard/pages/screening/callbacks.py [hunk 2]
```diff
@@ -654,94 +682,78 @@ def register_callbacks(elements, file_storage):
     )(functools.partial(upload_echo_data, file_storage=file_storage))
 
     callback(
-        Output("echo-bmg-combined", "data"),
+        Output("activation-inhibition-screening-options", "data"),
+        Output("activation-formula-dropdown", "disabled"),
+        Input("screening-feature-dropdown", "value"),
+        Input("activation-formula-dropdown", "value"),
+    )(functools.partial(on_additional_options_change))
```
Comment (Tremirre) [inline dashboard/pages/screening/callbacks.py:689]: same here - no need for functools.partial wrapper

File: dashboard/pages/screening/callbacks.py [hunk 3]
```diff
@@ -277,12 +280,37 @@ def upload_echo_data(
...
+    formula: str,
+) -> dict[str, str]:
+    """
+    Update the additional screening options dictionary
+
+    :param screening_feature: screening feature
+    :param formula: formula
+    :return: updated dictionary
+    """
+    disabled = False if key == "activation" else True
```
Comment (Tremirre) [inline dashboard/pages/screening/callbacks.py:294]: can be simplified to
`disabled = key != "activation`

**Review by Tremirre — APPROVED** [review]
looking good in general, added some minor comments

as a sidenote I think that the `Mode` enum can be deleted if its no longer used anywhere

**Review by zuzg — APPROVED** [review]
everything looks good

**Pull request merged**